File: android/src/main/java/org/ligi/passandroid/ui/views/CategoryIndicatorViewWithIcon.java
package org.ligi.passandroid.ui.views;

import android.content.Context;
import android.graphics.Bitmap;
import android.util.AttributeSet;
import android.widget.ImageView;
import butterknife.ButterKnife;
import org.ligi.passandroid.R;

public class CategoryIndicatorViewWithIcon extends BaseCategoryIndicatorView {

    private ImageView iconImageView;

    public CategoryIndicatorViewWithIcon(Context context, AttributeSet attrs) {
        super(context, attrs, R.layout.category_indicator);
        iconImageView = ButterKnife.findById(this, R.id.iconImageView);
    }

    public void setIcon(final Bitmap iconBitmap) {
        iconImageView.setImageBitmap(iconBitmap);
    }

}

File: android/src/main/java/org/ligi/passandroid/ui/views/CategoryIndicatorViewWithIcon.kt
package org.ligi.passandroid.ui.views

import android.content.Context
import android.graphics.Bitmap
import android.util.AttributeSet
import android.widget.ImageView
import org.ligi.passandroid.R

class CategoryIndicatorViewWithIcon(context: Context, attrs: AttributeSet) : BaseCategoryIndicatorView(context, attrs, R.layout.category_indicator) {

    fun setIcon(iconBitmap: Bitmap) {
        findViewById<ImageView>(R.id.iconImageView).setImageBitmap(iconBitmap)
    }

}
